cmd/admin.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	cfgParser "github.com/runatlantis/atlantis/server/config"
+	"github.com/runatlantis/atlantis/server/config/valid"
+	"github.com/runatlantis/atlantis/server/legacy"
+	"github.com/runatlantis/atlantis/server/logging"
+	"github.com/runatlantis/atlantis/server/neptune/admin"
+	adminconfig "github.com/runatlantis/atlantis/server/neptune/admin/config"
+	neptune "github.com/runatlantis/atlantis/server/neptune/temporalworker/config"
+)
+
+type Admin struct{}
+
+// NewServer returns the real Atlantis server object.
+func (a *Admin) NewServer(userConfig legacy.UserConfig, config legacy.Config) (ServerStarter, error) {
+	ctxLogger, err := logging.NewLoggerFromLevel(userConfig.ToLogLevel())
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to build context logger")
+	}
+
+	globalCfg := valid.NewGlobalCfg(userConfig.DataDir)
+	validator := &cfgParser.ParserValidator{}
+
+	// TODO: fill in values from globalCfg
```

Comment:
don't think this is necessary here, but would appreciate more context on what you mean